Gruntfile.js

```diff
@@ -260,12 +260,12 @@ module.exports = function(grunt) {
 				libraries: '**'
 			},
 
-			'library_custom_uglify_params': {
+			'library_custom_terser_params': {
```

Comment:
It would still be good to have an `uglifyjs` test scenario to ensure compatibility.